dev-test/account.test.js

```diff
@@ -0,0 +1,135 @@
+import path from "path"
+import {describe} from "yargs"
+import {
+  emulator,
+  init,
+  getAccountAddress,
+  getContractAddress,
+  deployContractByName,
+  getScriptCode,
+  executeScript,
+  sendTransaction,
+  mintFlow,
+  getFlowBalance,
+  shallRevert,
+  shallResolve,
+  shallPass,
+  shallThrow,
+  createAccount,
+  HashAlgorithm,
+  SignatureAlgorithm,
+} from "../src"
+import {playgroundImport} from "../src/transformers"
+import * as fcl from "@onflow/fcl"
+import {ec as EC} from "elliptic"
+import {validateKeyPair} from "./util/validate-key-pair"
+
+// We need to set timeout for a higher number, because some transactions might take up some time
+jest.setTimeout(10000)
+
+beforeEach(async () => {
+  const basePath = path.resolve(__dirname, "./cadence")
+  await init(basePath)
+  return emulator.start()
+})
+
+afterEach(async () => {
+  emulator.stop()
+})
+
+it("createAccount - should work with name and resolves to correct getAccountAddress", async () => {
+  const addr1 = await createAccount({
+    name: "Billy",
+  })
+
+  const addr2 = await getAccountAddress("Billy")
+
+  expect(addr1).toBe(addr2)
+})
+
+it("createAccount - should work without name and returns address", async () => {
+  const Billy = await createAccount({
+    name: "Billy",
+  })
+
+  expect(Billy).toMatch(/^0x[0-9a-f]{16}$/)
+})
+
+it("createAccount - should work with custom keys", async () => {
+  const privateKey = "1234"
+  const Billy = await createAccount({
+    name: "Billy",
+    keys: [
+      {
+        privateKey,
+        hashAlgorithm: HashAlgorithm.SHA2_256,
+        signatureAlgorithm: SignatureAlgorithm.ECDSA_secp256k1,
+      },
+      {
+        privateKey,
+      },
+    ],
+  })
+
+  expect(Billy).toMatch(/^0x[0-9a-f]{16}$/)
+
+  const keys = await fcl.account(Billy).then(d => d.keys)
+
+  expect(keys.length).toBe(2)
+  expect(keys[0].hashAlgoString).toBe("SHA2_256")
```

Comment:
Would be cool to have `SHA2_256` and others as constants you can import to test against.